### PR TITLE
Multi-platform tagging details

### PR DESCRIPTION
**Pull request by twaugh**
Signed-off-by: Tim Waugh <twaugh@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
cc @mlangsdorf

**Comment by twaugh** [comment]
Updated. I'll squash commits before merging.

File: docs/multiarch.rst [hunk 1]
```diff
@@ -801,6 +801,22 @@ ATOMIC_REACTOR_PLUGINS environment variable for an orchestrator build.
...
       },
       {
+        "name": "tag_from_config",
+        "args": {
+          "tags": {
+            "unique": [
+              "target-20170123055916"
+            ],
+            "primary": [
+              "$version-$release",
```
Comment (lcarva) [inline docs/multiarch.rst]: Should this be `'{version}-{release}'` instead?

**Comment by twaugh** [comment]
Updated.

**Comment by twaugh** [comment]
Updated.

File: docs/multiarch.rst [hunk 2]
```diff
@@ -919,13 +935,67 @@ signatures), and will fail if there are any mismatches.
...
+osbs-client will populate with the unique and primary tag suffixes to
+use.
+
+It will perform parameter substitution on the primary tag suffixes,
+using labels defined in the Dockerfile (in combination with the
+environment variables from the parent image).
+
+For the orchestrator build it will be given:
+- a unique tag suffix (without a platform name)
+- the strings ``$version-$release``, ``$version`` and ``latest``
```
Comment (lcarva) [inline docs/multiarch.rst]: Still says `$` based strings.

**Comment by twaugh** [comment]
Updated.

**Pull request merged**